### PR TITLE
Bump z2jh to v1.0 beta

### PR DESCRIPTION
**Pull request by yuvipanda**
- Move `singleuser.admin` to `custom.singleuserAdmin` to satisfy the helm
  JSON schema, which doesn't allow extra properties under `singleuser`
- Bump our hub base image to inherit from the v1.0-beta1 image
- Merge `requirements.yaml` file into `Chart.yaml` files - with helm3 these 
   do not need to be together
- Add couple more things to .gitignore

Splitting up code from https://github.com/2i2c-org/pilot-hubs/pull/429 that
works on just upgrading code to z2jh 1.0-beta1

Fixes https://github.com/2i2c-org/pilot-hubs/issues/414

### CODE REVIEW

**Review by consideRatio — APPROVED** [review]
Thanks @yuvipanda this LGTM!

I saw that you have pushed the k8s-hub replacement image as well. Merge?

File: hub-templates/basehub/values.yaml [hunk 1]
```diff
@@ -67,8 +72,6 @@ jupyterhub:
     hook:
       enabled: false
   proxy:
-    https:
-      enabled: false
```
Comment (damianavila) [inline hub-templates/basehub/values.yaml:71]: I guess this is now `false` by default?

File: hub-templates/daskhub/values.yaml [hunk 2]
```diff
@@ -41,7 +41,6 @@ basehub:
               - port: 22
                 protocol: TCP
       cloudMetadata:
-        enabled: true
```
Comment (damianavila) [inline hub-templates/daskhub/values.yaml:44]: And this is now `true` by default, correct?

File: hub-templates/basehub/values.yaml (hunk 1 above)
Comment (consideRatio) [reply hub-templates/basehub/values.yaml:71]: Yepp!

File: hub-templates/daskhub/values.yaml (hunk 2 above)
Comment (consideRatio) [reply hub-templates/daskhub/values.yaml:44]: No it renamed to the other field below called blockWithIpTables or something, and inverted its value. Having this set leads to an error as the helm charts forces a migration. There is no actual change in outcome by this change now.

File: hub-templates/basehub/values.yaml [hunk 3]
```diff
@@ -29,10 +29,15 @@ nfsPVC:
...
 jupyterhub:
   custom:
+    singleuserAdmin:
+      extraVolumeMounts:
+        - name: home
+          mountPath: /home/jovyan/shared-readwrite
+          subPath: _shared
     cloudResources:
-      provider: null
+      provider:
```
Comment (GeorgianaElena) [inline hub-templates/basehub/values.yaml:38]: Is there any difference between leaving this black and not setting them at all? Or is this a way to unset default values?

File: hub-templates/basehub/values.yaml (hunk 1 above)
Comment (yuvipanda) [reply hub-templates/basehub/values.yaml:71]: Yeah but it was also repeated down in the same file

File: hub-templates/basehub/values.yaml (hunk 3 above)
Comment (yuvipanda) [reply hub-templates/basehub/values.yaml:38]: ruamel.yaml formatted it this way, I assume it's equivalent?

File: hub-templates/basehub/values.yaml (hunk 1 above)
Comment (damianavila) [reply hub-templates/basehub/values.yaml:71]: Gotcha!

File: hub-templates/daskhub/values.yaml (hunk 2 above)
Comment (damianavila) [reply hub-templates/daskhub/values.yaml:44]: OK, thanks for the additional info!

**Review by damianavila — APPROVED** [review]
My questions were answered, this LGTM now!

**Comment by yuvipanda** [comment]
Thanks, @damianavila!

**Pull request merged**